comm_norms.adoc

```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
 === When issues become the subject of heated debate
 
 At times, the details of addressing a particular issue can cause tension or argument within the community.
+PD--"Argument" should either be made plural, or "an" should be inserted before it.
+
 While healthy and respectful debate is part of any thriving project—software or otherwise—tempers can flare easily, and (as has been well documented) people tend to behave with less civility online than they would in person.
+PD--What's your source for "as has been well documented"?
```

Comment:
https://www.bbc.com/future/article/20180403-why-do-people-become-trolls-online-and-in-social-media